bin/git-forgit

```diff
@@ -70,6 +70,10 @@ hash emojify &>/dev/null && _forgit_emojify='|emojify'
 
 # extract the first git sha occurring in the input and strip trailing newline
 _forgit_extract_sha="grep -Eo '[a-f0-9]+' | head -1 | tr -d '[:space:]'"
+_forgit_extract_sha() {
```

Comment:
This can be done and I removed the `_forgit_extract_sha` function in ab5a9e1. Unfortunately we need to ignore a lot of shellchecks and introduce more usage of `eval` to do so, which I am not happy about. From my perspective it would be better to keep the function in this case. I'll leave the decision to you, if you want to keep the changes from ab5a9e1 I will squash them into the previous commit, otherwise I will remove them. WDYT?

Reply:
Ah, I see. Well, then let's remove it again. Thanks for trying, though!